yarn-project/end-to-end/src/uniswap_trade_on_l1_from_l2.test.ts

```diff
@@ -14,16 +14,20 @@ import { CheatCodes } from './cheat_codes.js';
 import { CrossChainTestHarness } from './cross_chain/test_harness.js';
 import { delay, deployAndInitializeNonNativeL2TokenContracts, setup } from './utils.js';
 
-// PSA: this works on a fork of mainnet but with the default anvil chain id. Start it with the command:
+// PSA: This tests works on forked mainnet. There is a dump of the data in `dumpedState` such that we
+// don't need to burn through RPC requests.
+// To generate a new dump, use the `dumpChainState` cheatcode.
+// To start an actual fork, use the command:
 // anvil --fork-url https://mainnet.infura.io/v3/9928b52099854248b3a096be07a6b23c --fork-block-number 17514288 --chain-id 31337
 // For CI, this is configured in `run_tests.sh` and `docker-compose.yml`
 
+const dumpedState = 'src/dumps/uniswap_state';
+const EXPECTED_FORKED_BLOCK = 0; //17514288;
```

Comment:
nit - as a comment on how dumped state doesn't change block number but it works nevertheless